docs/src/main/asciidoc/spring-cloud-gateway.adoc

```diff
@@ -1660,7 +1660,7 @@ spring:
   cloud:
     gateway:
       globalcors:
-        corsConfigurations:
```

Comment:
I would suggest updating the test also if we make this change.

Reply:
Which test specifically?

This only changes the doc snippet to match the style of what is in the config metadata/appendix. In terms of functionality, dash-separated and camelCase [should both work](https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-external-config.html#boot-features-external-config-relaxed-binding), but 1.) consistent style is better and 2.) IntelliJ would say that `corsConfiguration` property cannot be resolved (since it does not match the metadata file).

Reply:
See here: https://github.com/spring-cloud/spring-cloud-gateway/blob/master/spring-cloud-gateway-core/src/test/resources/application.yml#L14

Reply:
Got it. Thanks!